server/manager.py

```diff
@@ -231,16 +240,21 @@ def start(args, args_string):
     Launch python server.
     """
     print("Launching Server")
+    """
     cwd = os.getcwd().split("CallFlow")[0] + "CallFlow/server/main.py"
     server_cmd = ["python3", cwd] + args_string
+    """
+    server_cmd = ["callflow_server"] + args_string
     launch_cmd(server_cmd, alias="server")
 
     """
     Launch callflow app server.
     """
     print("Launching client")
-    cwd = os.getcwd().split("CallFlow")[0] + "CallFlow/app"
-    prefix_string = ["--silent", "--prefix=" + cwd]
+    # cwd = os.getcwd().split("CallFlow")[0] + "CallFlow/app"
```

Comment:
Can we remove the commented statements? Its probably good to not accumulate old code.

Reply:
yes